### PR TITLE
[REVIEW][FEATURE][UWANG-219] - Screen create password admin

### PR DESCRIPTION
**Pull request by fahmi-azzuhri**
[REVIEW][FEATURE][UWANG-219] - Screen create password admin

#### Description

> Description of features, bugs, improvements or any changes in this MR.

#### Ticket Reference(s)

[UWANG-219](https://bluehabit.atlassian.net/browse/UWANG-219)

##### UI Changes

> Screenshot or video of UI-related changes
localhost-3000-access-management-admin-signup

##### Coverage

> Test coverage screenshot.

N/A

#### Note(s)

> Mention other user or extra note unrelated to the MR here.

[UWANG-219]: https://bluehabit.atlassian.net/browse/UWANG-219?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[UWANG-219]: https://bluehabit.atlassian.net/browse/UWANG-219?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: components/CardAdminSignup.vue
Comment (azkifairuz) [inline components/CardAdminSignup.vue:1]: izin saran mi, di bagian card ini tambahin emit buat triger function ubah passwordnya nya nnti pas componentnya dipanggil di screen nya, contohnya bisa liat di component sidebar ya mi,sisanya udah aman mi gg

File: pages/access-management/admin-signup/index.vue
Comment (azkifairuz) [inline pages/access-management/admin-signup/index.vue:1]: kalo niatnya cmn mindahin logic, masih bisa ko mi si cardnya dijadiin component nnti di component card pake defineEmit

File: components/CardAdminSignup.vue
Comment (fahmi-azzuhri) [reply components/CardAdminSignup.vue:1]: Ok ok ki tq

File: pages/access-management/admin-signup/index.vue
Comment (fahmi-azzuhri) [reply pages/access-management/admin-signup/index.vue:1]: Tq ki saran nya nanti balik kerja diperbaiki

File: components/CardAdminSignup.vue
Comment (fahmi-azzuhri) [reply components/CardAdminSignup.vue:1]: baru ngeh ki, di screen signup admin kan gaada ubah password nya

Comment (azkifairuz) [reply components/CardAdminSignup.vue:1]: oalah  giitu, oke mi

**Pull request merged**